entities/entities/lab_base/init.lua

```diff
@@ -34,12 +34,15 @@ end
 function ENT:Destruct()
     local vPoint = self:GetPos()
 
-    util.BlastDamage(self, self, vPoint, 200, 200)
+    util.BlastDamage(self, self, vPoint, 100, 100)
```

Comment:
Why reduce the radius?

Reply:
> Why reduce the radius?

> * because it previously was extremely high 🙃

Reply:
Oh sorry, didn't see the comment.
The radius also was very high.
100 should be alright, especially because it deals 100 damage.

Reply:
That's a nasty way to push an opinion. This pr is about adding effects. Side loading a change in radius is rude.

Reply:
It's a like 5 line code change and I mentioned damage being adjusted.
I wasn't trying to silently push a change, this isn't my first pull request, nor my first issue.

Reply:
Oh, sorry. I missed that.